samples/create-buffers-around-points/src/main/java/com/esri/arcgismaps/sample/createbuffersaroundpoints/components/CreateBuffersAroundPointsViewModel.kt

```diff
@@ -0,0 +1,291 @@
+                    messageDialogVM.showMessageDialog("Error projecting boundary geometry")
+                    return@launch
+                }
+
+                val builder = PolygonBuilder(spatialReference = statePlaneNorthCentralTexas)
+                polygonGeoms.filterNotNull().forEach { builder.addPoint(it) }
+                boundaryPolygon = builder.toGeometry()
+
+                // The boundary polygon extent to configure the map's initial viewpoint
+                boundaryPolygon?.let { poly ->
+                   // Create and add a dashed red boundary graphic
+                    val boundaryOutline = SimpleLineSymbol(
+                        style = SimpleLineSymbolStyle.Dash,
+                        color = Color.red,
+                        width = 5f
+                    )
+                    val boundaryGraphic = Graphic(geometry = poly, symbol = boundaryOutline)
+                    boundaryGraphicsOverlay.graphics.add(boundaryGraphic)
+                }
+
+                // Configure buffer overlay renderer (yellow fill with green outline)
+                val bufferOutline = SimpleLineSymbol(
+                    style = SimpleLineSymbolStyle.Solid,
+                    color = Color.green,
+                    width = 3f
+                )
+                val bufferFill = SimpleFillSymbol(
+                    style = SimpleFillSymbolStyle.Solid,
+                    color = Color.fromRgba(r = 255, g = 255, b = 0, a = 153),
+                    outline = bufferOutline
+                )
+                bufferGraphicsOverlay.renderer = SimpleRenderer(symbol = bufferFill)
+
+                // Configure tapped points overlay renderer (small red circles)
+                val tapSymbol = SimpleMarkerSymbol(
+                    style = SimpleMarkerSymbolStyle.Circle,
+                    color = Color.red,
+                    size = 10f
+                )
+                tappedPointsGraphicsOverlay.renderer = SimpleRenderer(symbol = tapSymbol)
+
+                //Set initial viewpoint of the map
+                arcGISMap.initialViewpoint = Viewpoint(boundingGeometry = boundaryPolygon as Geometry)
+                // Load the map; report any failure
+                arcGISMap.load().onFailure { messageDialogVM.showMessageDialog(it) }
+
+
+            } catch (e: Throwable) {
+                messageDialogVM.showMessageDialog(e)
+            }
+        }
+    }
+
+    /**
+     * Called by the MapView when the user taps the map. If the tap is within the valid boundary,
+     * request a buffer radius input from the UI. Otherwise update status to indicate out-of-bounds.
+     */
+    fun onMapTapped(mapPoint: Point) {
+        viewModelScope.launch {
+            val boundary = boundaryPolygon
+            if (boundary == null) {
+                _statusText.value = "Boundary not initialized"
+                return@launch
+            }
+
+            val contains = GeometryEngine.contains(boundary, mapPoint)
```

Comment:
Recommendation to use named args: 

```suggestion
            val contains = GeometryEngine.contains(geometry1 = boundaryPolygon, geometry2 = mapPoint)
```